xchange-stream-kraken/src/main/java/info/bitrich/xchangestream/kraken/KrakenStreamingService.java

```diff
@@ -285,4 +316,20 @@ static Integer parseOrderBookSize(Object[] args) {
     }
     return null;
   }
+
+  public static void main(String[] args)
```

Comment:
why are we adding a main here?

Reply:
I'm sorry, committed a method that I used for testing. Should be OK now